proto/build.rs

```diff
@@ -64,6 +64,27 @@ static CUSTOM_FIELD_ATTRIBUTES: &[(&str, &str)] = &[
 ];
 
 fn main() -> Result<()> {
+    println!("protox::compile_fds...");
```

Comment:
I think it'll be visible with enough verbose flags passed, or in case of failed build. 
I initially thought it was some kind of custom build instruction, [similar to `cargo::`](https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script) ones, but I don't believe it's that? 
Either way I think output from protox/proto build should be enough, +1 for removing it.